src/syscheckd/syscheck.h

```diff
@@ -84,6 +84,13 @@ typedef struct diff_data {
     char *diff_file;
 } diff_data;
 
+typedef enum audit_key_type {
+    FIM_AUDIT_UNKNOWN_KEY = 0,
+    FIM_AUDIT_KEY,
+    FIM_AUDIT_HC_KEY,
+    FIM_AUDIT_CUSTOM_KEY
+} audit_key_type;
+
```

Comment:
Could we move this enum to `syscheck_audit.h`? Since it is only used within the whodata directory, I think it makes more sense to limit its scope a little.